dts/bindings/stepper/adi/adi,tmc2130.yaml

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Navimatix GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Analog Devices TMC2130 stepper motor driver.
+  Implements only the Step Dir Interface with SPI driver configuration. The Standalone and
+  Spi Direct Driver Interfaces are not supported.
```

Comment:
> The Standalone and Spi Direct Driver Interfaces are not supported.

This is redundant. If I remember it correctly, I asked you to specify the mode which this supports, now what it doesn't.